src/transformers/utils/fx.py

```diff
@@ -616,6 +671,20 @@ def _proxies_to_metas(v):
     return v
 
 
+def _patch_fn_for_proxy(fn, tracer):
```

Comment:
Why is this needed compared to what is being done in `HFTracer.create_proxy`?

Reply:
Discussed offline